src/strategies/GetEventNameStrategy.ts

```diff
@@ -0,0 +1,24 @@
+import i18next from 'i18next';
+import { Context } from 'telegraf';
+import { ChatData } from 'Bot';
+import Strategy from 'strategies/Strategy';
```

Comment:
it is ok to leave, but I feel very weird distinguishing what is an npm module what is not